frontend/src/tabs/zoom.pug

```diff
@@ -56,8 +56,9 @@
       h3(v-else) {{ day.date }}
       template
         .commit-message(
-            v-for="slice in day.commitResults"
-            v-bind:key="slice.hash"
+            tabindex="-1",
```

Comment:
May I now the usage of this line?

Reply:
To enable focus for div

Reply:
maybe inline a comment then, so that other devs will be reminded what it is for